backend/app/routes/Q&A/question/updateQuestionStatus.js

```diff
@@ -0,0 +1,44 @@
+const to  = require("await-to-js").default;
+
+const constants = require("../../../../constants");
+const { ErrorHandler } = require("../../../../helpers/error");
+
+const question = require("../../../models/question");
+
+module.exports = async (req, res, next) => {
+    // getting id and status from body
+    const id = req.body.id;
+    const status = req.body.status
+
+    // query fro updating
+    const [ err, result ] = await to(question.findOneAndUpdate({ _id : id }, { $set : { isApproved : status } }));
+
+    // error occured due to the some problem
+    if(err) {
+        const Error = new ErrorHandler(constants.ERRORS.DATABASE, {
+            statusCode: 500,
+            message: 'Database Error',
+            errStack: err,
+        });
+
+        return next(Error);
+    }
+
+    // if result is null that means question with given id is not exist in collection
+    if(result === null) {
+        const Error2 = new ErrorHandler(constants.ERRORS.INPUT, {
```

Comment:
Can we please give meaningful name here? Also, please use `cameCase` for variable name like `error`

Reply:
Fixed